assemblyline_core/tasking_client.py

```diff
@@ -60,6 +60,13 @@ def __init__(self, datastore: AssemblylineDatastore = None, filestore: FileStore
             self.cleanup = True
         self.identify = identify or forge.get_identify(config=self.config, datastore=self.datastore, use_cache=True)
 
+        try:
```

Comment:
Why does this has to be in a try catch block? don't we just want it to crash hard if that fails anyway?